agent/acs/handler/acs_handler.go

```diff
@@ -194,6 +197,8 @@ func NewSession(
 // If the instance is deregistered, Start() would emit an event to the
 // deregister-instance event stream and sets the connection backoff time to 1 hour.
 func (acsSession *session) Start() error {
```

Comment:
nit. Let's update the comments with changes made on this PR for this function!

Reply:
Got it, thank you!